test/index.spec.js

```diff
@@ -7,52 +7,52 @@ const ipfsApi = require('ipfs-api')
-  this.timeout(2000)
-  let Node
-
-  it('has the correct path when installed with npm3', (done) => {
-    process.env.testpath = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/lib' // fake __dirname
-    let npm3Path = '/tmp/ipfsd-ctl-test/node_modules/go-ipfs-dep/go-ipfs'
-
-    mkdirp(npm3Path, (err) => {
-      if (err) {
-        throw err
-      }
-
-      fs.writeFileSync(path.join(npm3Path, 'ipfs'))
-      delete require.cache[require.resolve('../src/node.js')]
-      Node = require('../src/node.js')
-      var node = new Node()
-      assert.equal(node.exec, '/tmp/ipfsd-ctl-test/node_modules/go-ipfs-dep/go-ipfs/ipfs')
-      rimraf('/tmp/ipfsd-ctl-test', done)
-    })
-  })
-
-  it('has the correct path when installed with npm2', (done) => {
-    process.env.testpath = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/lib' // fake __dirname
-    let npm2Path = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/node_modules/go-ipfs-dep/go-ipfs'
-
-    mkdirp(npm2Path, (err) => {
-      if (err) {
-        throw err
-      }
-
-      fs.writeFileSync(path.join(npm2Path, 'ipfs'))
-      delete require.cache[require.resolve('../src/node.js')]
-      Node = require('../src/node.js')
-      var node = new Node()
-      assert.equal(node.exec, '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/node_modules/go-ipfs-dep/go-ipfs/ipfs')
-      rimraf('/tmp/ipfsd-ctl-test', done)
-    })
-  })
-})
+// describe('ipfs executable path', function () {
+//   this.timeout(2000)
+//   let Node
+//
+//   it('has the correct path when installed with npm3', (done) => {
+//     process.env.testpath = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/lib' // fake __dirname
+//     let npm3Path = '/tmp/ipfsd-ctl-test/node_modules/go-ipfs-dep/go-ipfs'
+//
+//     mkdirp(npm3Path, (err) => {
+//       if (err) {
+//         throw err
+//       }
+//
+//       fs.writeFileSync(path.join(npm3Path, 'ipfs'))
+//       delete require.cache[require.resolve('../src/node.js')]
+//       Node = require('../src/node.js')
+//       var node = new Node()
+//       assert.equal(node.exec, '/tmp/ipfsd-ctl-test/node_modules/go-ipfs-dep/go-ipfs/ipfs')
+//       rimraf('/tmp/ipfsd-ctl-test', done)
+//     })
+//   })
+//
+//   it('has the correct path when installed with npm2', (done) => {
+//     process.env.testpath = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/lib' // fake __dirname
+//     let npm2Path = '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/node_modules/go-ipfs-dep/go-ipfs'
+//
+//     mkdirp(npm2Path, (err) => {
+//       if (err) {
+//         throw err
+//       }
+//
+//       fs.writeFileSync(path.join(npm2Path, 'ipfs'))
+//       delete require.cache[require.resolve('../src/node.js')]
+//       Node = require('../src/node.js')
+//       var node = new Node()
+//       assert.equal(node.exec, '/tmp/ipfsd-ctl-test/node_modules/ipfsd-ctl/node_modules/go-ipfs-dep/go-ipfs/ipfs')
+//       rimraf('/tmp/ipfsd-ctl-test', done)
+//     })
+//   })
+// })
```

Comment:
Yes because it's no longer up to npm where the ipfs binary is stored. Currently, unless you provide an IPFS_EXEC path, the path for binary defaults to `./vendor` under the module root directory. Let me know if you would prefer something else.